Core/Utilities/ThreadSafe.py

```diff
@@ -36,8 +35,8 @@ def unlock(self):
     return self.__lock.release()
 
 
-# FIXME: not used
-class WORM:
+# FIXME: not used UNUSED
```

Comment:
why not deleting it then ?

Reply:
I'll open a PR now for `integration` for removing this stuff.